src/plugins/creditor/credNodeStore.test.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+import {type CreditorNode} from "./creditorNode";
+import Store, {getDetails, toNodeEntry} from "./credNodeStore";
+import {random as randomUuid} from "../../util/uuid";
+
+describe("plugins/creditor/credNodeStore", () => {
+  const tagId1 = randomUuid();
+  const tagId2 = randomUuid();
```

Comment:
Speaking from experience: Using random UUIDs in test code will just create headaches, e.g. whenever we want to do snapshot serialization. I recommend something like the following:
https://github.com/sourcecred/sourcecred/blob/8ade386780654e69a10b134d67d2b9ac66ac1c79/src/ledger/ledger.test.js#L27-L43